security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,13 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-6-1_{context}"]
+== OpenShift Compliance Operator 1.6.1
+
+The following advisory is available for the OpenShift Compliance Operator 1.6.1:
+
+* link:https://access.redhat.com/errata/RHBA-2024:10367[RHBA-2024:10367 - OpenShift Compliance Operator update]
+
```

Comment:
Do we need to add anything about upgrading dependencies in the underlying base images?

Reply:
I'll add the statement we've used previously that these updates address upgraded dependencies in underlying images. 👍